t/ui/10-tests_overview.t

```diff
@@ -354,6 +354,33 @@ subtest 'filtering by module' => sub {
     };
 };
 
+subtest 'filtering by module_re' => sub {
+    my $module_re         = 'test-kate';
```

Comment:
I don't quite get the search approach here. "kate" is the name of the test module but the feature is about searching *within* the module content, so better search for something that does not appear in the filename. A realistic use case would be to search for the maintainer field. That would also make cdywan happy because you would search in the metadata. How about you search for something like `/Maintainer *okurz/`?

Reply:
ok, why not

Reply:
And maybe add one more test case with a slightly different regex that matches nothing.